From d85c547840d8debd155c821a388bcfa21e674114 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Wed, 27 Jul 2016 17:09:37 -0700
Subject: [PATCH] HEAD-2317 sdc-headnode "make check" should work without SSH
 keys Reviewed by: Alex Wilson <alex.wilson@joyent.com> Approved by: Alex
 Wilson <alex.wilson@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index ce8aee52..0eaf82cd 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "extsprintf": "^1.3.0",
     "jscs": "^1.13.1",
     "jshint": "^2.7.0",
-    "manta": "git+ssh://git@github.com:joyent/node-manta.git#12f4c84",
+    "manta": "git+https://github.com/joyent/node-manta.git#12f4c84",
     "monowrap": "^1.0.4",
     "progbar": "^1.1.0",
     "readtoend": "0.0.1",
-- 
2.21.0

